### PR TITLE
first working version of epub_loader

### PR DESCRIPTION
**Pull request by danielmachado99**

### CODE REVIEW

File: backend/loaders/epub_loader.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+from tempfile import SpooledTemporaryFile
+from io import BytesIO, UnsupportedOperation
+from pathlib import Path
+import shutil
+def load_epub(file: bytes, filename: str, file_meta: UploadFile):
+    misc_dir = os.path.join(os.getcwd(), "misc")
+    
+    file_location = f"{misc_dir}/{filename}.txt"
+    with open(file_location, 'w') as fp:
+        pass
```
Comment (grumpyp) [inline backend/loaders/epub_loader.py:15]: needs fix

File: backend/loaders/epub_loader.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    with open(file_location, "wb+") as file_object:
+        shutil.copyfileobj(file, file_object)
+    book = epub.read_epub(file_location)
+
+    text=""
+    for doc in book.get_items_of_type(ebooklib.ITEM_DOCUMENT):
+       
+        text=text + BeautifulSoup(doc.content).get_text()
+    with open(f"{misc_dir}/{filename}.txt", "w") as f:
+        f.write(text)
```
Comment (dipta10) [inline backend/loaders/epub_loader.py:25]:
```suggestion
    contents = []
    for doc in book.get_items_of_type(ebooklib.ITEM_DOCUMENT):
        content = BeautifulSoup(doc.content).get_text().strip()
        content = content.replace("\n", " ")
        contents.append(content)
    
    text = "\n".join(contents)
```
- we need to replace the new lines
- and I think maybe concatenating immutable strings in the loop might be costly. In that case, we can use join

Comment (danielmachado99) [reply backend/loaders/epub_loader.py:25]: Will make the required changes. I dont think we need to replace "\n". If we do then the output textfile will be one continuous text file without anything separating what should be newlines

File: backend/loaders/epub_loader.py (hunk 1 above)
Comment (danielmachado99) [reply backend/loaders/epub_loader.py]: Have removed this now.

File: backend/loaders/epub_loader.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+# print(book, "book")
+
+
+# for doc in book.get_items_of_type(ebooklib.ITEM_DOCUMENT):
+#     print("start reading*****")
+#     text=BeautifulSoup(doc.content).get_text()
+#     print("end reading")
+#     break
+
+# print(text)
```
Comment (grumpyp) [inline backend/loaders/epub_loader.py:40]: delete these comments pls

File: backend/requirements.txt
Comment (grumpyp) [inline backend/requirements.txt:1]: are all these new modules nessesary?

**Comment by grumpyp** [comment]
done in #60

**Pull request closed without merging**